packages/ember-htmlbars/tests/helpers/input_test.js

```diff
@@ -1,492 +0,0 @@
-  ok(view.$('input').is(':not(:disabled)'), 'There are no disabled text fields');
-});
-
-QUnit.test('input value is updated when setting value property of view', function() {
-  equal(view.$('input').val(), 'hello', 'renders text field with value');
-
-  let id = view.$('input').prop('id');
-
-  run(null, set, controller, 'val', 'bye!');
-  equal(view.$('input').val(), 'bye!', 'updates text field after value changes');
-
-  equal(view.$('input').prop('id'), id, 'the component hasn\'t changed');
-});
-
-QUnit.test('input placeholder is updated when setting placeholder property of view', function() {
-  equal(view.$('input').attr('placeholder'), 'Enter some text', 'renders text field with placeholder');
-  run(null, set, controller, 'place', 'Text, please enter it');
-  equal(view.$('input').attr('placeholder'), 'Text, please enter it', 'updates text field after placeholder changes');
-});
-
-QUnit.test('input name is updated when setting name property of view', function() {
-  equal(view.$('input').attr('name'), 'some-name', 'renders text field with name');
-  run(null, set, controller, 'name', 'other-name');
-  equal(view.$('input').attr('name'), 'other-name', 'updates text field after name changes');
-});
-
-QUnit.test('input maxlength is updated when setting maxlength property of view', function() {
-  equal(view.$('input').attr('maxlength'), '30', 'renders text field with maxlength');
-  run(null, set, controller, 'max', 40);
-  equal(view.$('input').attr('maxlength'), '40', 'updates text field after maxlength changes');
-});
-
-QUnit.test('input size is updated when setting size property of view', function() {
-  equal(view.$('input').attr('size'), '30', 'renders text field with size');
-  run(null, set, controller, 'size', 40);
-  equal(view.$('input').attr('size'), '40', 'updates text field after size changes');
-});
-
-QUnit.test('input tabindex is updated when setting tabindex property of view', function() {
-  equal(view.$('input').attr('tabindex'), '5', 'renders text field with the tabindex');
-  run(null, set, controller, 'tab', 3);
-  equal(view.$('input').attr('tabindex'), '3', 'updates text field after tabindex changes');
-});
-
-QUnit.test('cursor position is not lost when updating content', function() {
-  equal(view.$('input').val(), 'hello', 'precondition - renders text field with value');
-
-  var $input = view.$('input');
-  var input = $input[0];
-
-  // set the cursor position to 3 (no selection)
-  run(function() {
-    input.value = 'derp';
-    view.childViews[0]._elementValueDidChange();
-    input.selectionStart = 3;
-    input.selectionEnd = 3;
-  });
-  run(null, set, controller, 'val', 'derp');
-
-  equal(view.$('input').val(), 'derp', 'updates text field after value changes');
-
-  equal(input.selectionStart, 3, 'cursor position was not lost');
-  equal(input.selectionEnd, 3, 'cursor position was not lost');
-});
-
-QUnit.test('input can be updated multiple times', function() {
```

Comment:
moved to `@htmlbars Helpers test: {{input}}` : [`@test dynamic attributes`](https://github.com/emberjs/ember.js/pull/13194/files#diff-0caed08a992497ff104284ec6aea6d96R108)